### DISCUSSION TITLE
Introduce a way to know why your score doesn't count toward leaderboards

### DISCUSSION THREAD

**Discussion by gael-vanderlee** [monkeytypegame/monkeytype#3828] [Ideas] [answered]
Hello,

For some reason, my typing tests don't get me on the daily leaderboard. I am using standard english, no punctuatinon, 15sec etc. 
I might be missing a setting but finding which one is difficult and frustrating. Adding a button in the leaderboard to either tell you what you need to change or to change you to those settings for a single run would be a welcome addition.

**Comment by Miodec** [comment, marked as answer]
Daily leaderboard has a limited number of places, that's probably why you cant get on it.

**Comment by gael-vanderlee** [comment]
I see. It might be good to show this explicitely to the user. Right now it only says "Not qualified" on the leaderboard which can get confusing.
